lib/rubocop/cop/style/if_with_boolean_literal_branches.rb

```diff
@@ -8,6 +8,20 @@ module Style
       # The conditions to be checked are comparison methods, predicate methods, and double negative.
       # `nonzero?` method is allowed by default.
       # These are customizable with `AllowedMethods` option.
+      # This cop targets only `if`s with a single `elsif` or `else` branch.
+      #
+      # [source,console]
```

Comment:
Shouldn't it say `ruby` here instead of `console`?

Reply:
Oops! I've fixed.